tests/system/demos/churn/test_churn.py

```diff
@@ -1,10 +1,13 @@
+import pytest
+
 import mlrun
 
 from tests.system.base import TestMLRunSystem
 from tests.system.demos.base import TestDemo
 
 
 @TestMLRunSystem.skip_test_if_env_not_configured
+@pytest.mark.enterprise
```

Comment:
I suggest to add some comment mentioning what's the reason this can't run on enterprise, same for the other places
note that there is mpi-operator in the mlrun-kit so not sure why it's marked as enterprise